gnovm/cmd/gno/test.go

```diff
@@ -182,9 +182,9 @@ func execTest(cfg *testCfg, args []string, io *commands.IO) error {
 		cfg.rootDir = guessRootDir()
 	}
 
-	paths, err := gnoPackagesFromArgs(args)
+	paths, err := targetFromPatterns(args)
```

Comment:
can we not extend this to other places where we use `gnoPackagesFromArgs`? (`gno lint`, `gno build`)

Reply:
```suggestion
	paths, err := targetsFromPatterns(args)
```

Reply:
@moul Resolved-by: d16ed9ec1b491cb90bdf3b7356497668bf01e4da
Thanks!

@thehowl we can. Infact in plan to do that, but in a separate PR.

Reply:
@harry-hov 
What's the blocker for adding it in this PR?

Reply:
@harry-hov 
Can you please elaborate on `small`?
You seemed to have no problem making larger changes in other PRs, including this one:
```text
+----------------------------------------------------------------+-----------+
| title                                                          | additions |
+----------------------------------------------------------------+-----------+
| feat: implement `gno test -cover`                              | 2465      |
| feat: support gno.mod                                          | 1383      |
| feat: implement grc721                                         | 923       |
| feat: use `modfile` package to write modfile                   | 875       |
| feat(gno.land/cmd): auto load and sort pkgs from examples      | 661       |
| feat: implement `r/game/2048`                                  | 498       |
| feat: sanitize gno.mod                                         | 427       |
| feat: implement `gno mod init` command                         | 324       |
| feat: support `gno clean` just like `go clean`                 | 271       |
| feat(mod): fetch indirect dependencies                         | 220       |
| feat: implement `gno mod tidy`                                 | 181       |
| feat: precompile imports and fix `gnodev test`                 | 180       |
| feat: fix `gno test` for `_test.gno` and `_filetest.gno` files | 178       |
| feat: `gno test` support `/...` pattern                        | 175       |
| feat(gnomod): support parsing deprecated message               | 161       |
| feat: support replacing module with another module             | 155       |
| [RFC][WIP] feat: `gno test` support external pkgs              | 119       |
| refactor(gnolang): remove unnecessary if-else                  | 117       |
| feat: addpkg command respects gno.mod file                     | 106       |
+----------------------------------------------------------------+-----------+
```

I just do not understand what the problem is with making the small change in this PR?

Reply:
Sorry I may have used word. Maybe `concise`? Im not sure whats the word. Please focus on the other line `This PR is focused on gno test` 🙏 

Since you mentioned my other PRs. I try to break down my PR into parts. As much as i can. Tried the same with my other PRs.
Please go through them let me know if they can be break down into smaller one. I will be more then happy to implement your feedbacks in my future PRs. Thanks.

Reply:
Resolving this conversation because I do not see any will to implement the most basic changes requested by fellow peers, or even understand why they are being suggested in the first place.

I don't believe the mentality of `will fix it later` is a good one for engineering teams, especially for a project that has accumulated an insane amount of technical debt already. In the end it is your PR, so you get the final say, just thought I'd leave my 2 cents on the issue